ru.bmstu.rk9.rao.lib/src/ru/bmstu/rk9/rao/lib/varconst/VarConst.java

```diff
@@ -0,0 +1,15 @@
+package ru.bmstu.rk9.rao.lib.varconst;
+
+import ru.bmstu.rk9.rao.lib.lambdaexpression.LambdaExpression;
+
+public abstract class VarConst {
+	private Double	start;
+	private Double	stop;
+	private Double	step;
+	private LambdaExpression lambda;
+	
+	public abstract String getName();
+	
+//	checkValue method creates in VarConstCompiler.xtend
+//	and getAllDependencies method too
```

Comment:
you can remove this comment now 
```suggestion
	public abstract boolean checkValue(HashMap<String, Double> args);
```
and replace it with abstract method